types/hooks/screens/useChatUtilsHook.types.ts

```diff
@@ -3,10 +3,13 @@ import {ChannelList} from '../../database/schema/ChannelList.types';
 interface UseChatUtilsHook {
   selectedChannel: ChannelList | null;
   goBack: () => void;
-  goToChatScreen: (channel: ChannelList) => void;
+  goToChatScreen: (channel: ChannelList, from?: string) => void;
   goToPostDetailScreen: (channel: ChannelList) => void;
+  goToCommunityScreen: (channel: ChannelList) => void;
   goToChatInfoScreen: () => void;
   goBackFromChatScreen: () => void;
+  handleTextSystem: (item: any) => string;
+  splitSystemMessage: (message: string) => string[];
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New functions have been added to the `UseChatUtilsHook` interface. Ensure that these new functions are implemented wherever this interface is used, and their usage is correctly handled. 

```diff
+   goToCommunityScreen: (channel: ChannelList) => void;
+   handleTextSystem: (item: any) => string;
+   splitSystemMessage: (message: string) => string[];
```
These changes seem fine as long as they are properly implemented and used.


<!-- This is an auto-generated comment by OSS CodeRabbit -->